### DISCUSSION TITLE
Example of radial division configuration

### DISCUSSION THREAD

**Discussion by jbpassot** [stella-cv/stella_vslam#424] [Q&A] [answered]
Hi, 

I am trying to use different camera model to understand the pros and cons of each different model for the same camera setup.
I got the system working well using a Theta Z1 camera using monocular equirectangular output and I would like now to try to get the same result using the raw output from the camera. My understanding is that the model to use in this particular case (raw unprocessed data) is the radial_division model, but I can't figure out which parameter to use in the yaml file. The documentation states that the distortion parameter should be set and does not talk about other parameters, but I could not find any example using this camera model.

Few questions on how to use the radial_diffusion camera model, more specifically for the Theta Z1 camera:

- Is there an example yaml file for the theta Z1 (2 lenses) using the radial_division model already available somewhere ?
- Do the fx, fy, cx, cy k1, k2, k3, k4 parameters need to be defined for the Z1 camera if using the "radial_diffusion" model ? 
- How to find the parameters for distortion ? 
- Does the radial_division model assume that each frame of the MP4 file contains the back and front camera images (back camera on the left, front camera on the right of the frame) or am I making a wrong assumption ?

Thanks in advance for your help ! 

JB

**Comment by ymd-stella** [comment, marked as answer]
See https://github.com/xdspacelab/openvslam/pull/282.
